Networking/Networking/Remote/ProductVariationsRemote.swift

```diff
@@ -114,6 +115,31 @@ public class ProductVariationsRemote: Remote, ProductVariationsRemoteProtocol {
         }
     }
 
+    /// Updates a specific `ProductVariations`.
+    ///
+    /// - Parameters:
+    ///     - productVariations: the ProductVariations to update remotely.
+    ///     - completion: Closure to be executed upon completion.
+    ///
+    public func updateProductVariations(productVariations: [ProductVariation], completion: @escaping (Result<[ProductVariation], Error>) -> Void) {
+        guard let siteID = productVariations.first?.siteID, let productID = productVariations.first?.productID else {
```

Comment:
I think it is more clear, explicit, and safe to pass the `siteID` and `productID` as function parameterers.

Reply:
I agree, it makes the API more robust, and less likely to be used in the wrong way. I fixed it!